marimo/_data/sql_summaries.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
+
+from typing import List, Tuple
+
+from marimo._data.get_datasets import _db_type_to_data_type
+from marimo._data.models import ColumnSummary, DataType
+from marimo._sql.sql import _wrapped_sql
+
+
+def get_sql_summary(
```

Comment:
Is there a way to check if the table is already loaded into `duckdb`, or if it's e.g. from Postgres? I think this could be very expensive for large Postgres/row-oriented tables

Reply:
Yes we could limit to in-memory